{
    "type": "bundle",
    "id": "bundle--82396315-b349-4b5d-a7d5-31eea7689a4d",
    "spec_version": "2.0",
    "objects": [
        {
            "created_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
            "object_marking_refs": [
                "marking-definition--fa42a846-8d90-4e51-bc29-71d5b4802168"
            ],
            "source_ref": "malware--58c5a3a1-928f-4094-9e98-a5a4e56dd5f3",
            "target_ref": "attack-pattern--b3d682b6-98f2-4fb0-aa3b-b4df007ca70a",
            "external_references": [
                {
                    "source_name": "Arxiv Avaddon Feb 2021",
                    "url": "https://arxiv.org/pdf/2102.04796.pdf",
                    "description": "Yuste, J. Pastrana, S. (2021, February 9). Avaddon ransomware: an in-depth analysis and decryption of infected systems. Retrieved August 19, 2021."
                }
            ],
            "description": "[Avaddon](https://attack.mitre.org/software/S0640) has used encrypted strings.(Citation: Arxiv Avaddon Feb 2021)",
            "relationship_type": "uses",
            "id": "relationship--cc134d2d-64eb-469a-b8eb-90ea632c0adc",
            "type": "relationship",
            "modified": "2021-08-23T19:38:33.525Z",
            "created": "2021-08-23T19:38:33.525Z"
        }
    ]
}